interfaces/builtin/network_manager.go

```diff
@@ -67,6 +67,29 @@ network packet,
 
 /dev/rfkill rw,
 
+# Needed for modem connections using PPP
+/usr/sbin/pppd ix,
+/etc/ppp/** rwix,
+/dev/ppp rw,
+/dev/tty[^0-9]* rw,
+/run/udev/data/* r,
+/proc/sys/kernel/modprobe r,
```

Comment:
ppp_deflate, ppp_async, maybe more depending on protocols (VPN case?). Sending packages through PPP and doing some tasks is at the kernel level due to efficiency reasons (otherwise you would need to get IP packets on user side and send them through the tty device back to the kernel).